index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Microverse</title>
+    <title>Microverse.</title>
 </head>
 <body>
     <h1>Hellow Microverse</h1>
```

Comment:
## 4.
### Your Linters
![linters-are-not-passing](https://user-images.githubusercontent.com/93039359/186142994-a15598d9-ad14-4eb1-bdd5-422ebdd7b827.png)


- [ ] Your linters are not passing all the tests. :cry: Please fix stylint errors from your code. These errors can break your application so please take them seriously :relaxed:

Reply:
I honestly need help with this linters. I don't understand the errors